backend/api/Services/MissionDefinitionService.cs

```diff
@@ -179,25 +187,75 @@ public async Task<MissionDefinition> Update(MissionDefinition missionDefinition)
             }
         }
 
+        public async Task<MissionDefinition> FindExistingOrCreateCustomMissionDefinition(CustomMissionQuery customMissionQuery, List<MissionTask> missionTasks)
```

Comment:
This function seems to already exist in `CustomMissionSchedulingService.cs`

Reply:
This was removed in a rebase so no longer present.

Reply:
This was still left there. Removed the unused one in MissionDefinitionService